TFT/src/User/API/interfaceCmd.c

```diff
@@ -359,7 +344,8 @@ void sendQueueCmd(void)
                   Serial_Puts(SERIAL_PORT_2, "\n");
                 }
                 Serial_Puts(SERIAL_PORT_2, "ok\n");
-                purgeLastCmd();
+                purgeLastCmd(avoid_terminal);
+
```

Comment:
no blank line was used for "return" in interfaceCmd. So it should be removed here